wiremock/mappings/top_state_no_data_about_to_submit.json

```diff
@@ -0,0 +1,15 @@
+{
+  "request": {
+    "method": "POST",
+    "urlPath": "/case_type/top/state/no_data/about_to_submit"
+  },
+  "response": {
+    "status": 200,
+    "headers": {
```

Comment:
do we need to introduce this file? Can't we test this corner scenario with a unit test rather than an integration test?

Reply:
We have already test it in a the integration test . But I think that it would be nice having this scenario in the stubs for further use and to complete the whole test coverage.

Reply:
it is already cover in the unit test.  Nonetheless I think that it would be nice having this test to cover all scenarios in the integration test.